pkg/controller/pravega/pravega_segmentstore.go

```diff
@@ -145,9 +157,11 @@ func makeSegmentstorePodSpec(p *api.PravegaCluster) corev1.PodSpec {
 func MakeSegmentstoreConfigMap(p *api.PravegaCluster) *corev1.ConfigMap {
 	javaOpts := []string{
 		"-Xms1g",
+		"-XX:MaxDirectMemorySize=1g",
```

Comment:
Can we make this configurable somehow? It would be very helpful for us if we could tweak this on the go via a manifest of some other way that doesn't include redeploying the operator.

Reply:
Thanks @andreipaduroiu , we should do that. This is also related to this [issue](https://github.com/pravega/pravega-operator/issues/226)

Reply:
@Tristan1900 lets take up support for java options as part of a separate PR.

Reply:
agree

Reply:
> @RaulGracia Thanks for the comment. You are right about the dup dump files, I didn't consider that situation before. Currently, I'm not sure how to make heap dump files having different names. Is it ok to just maintain one?
> 
> About raising another issue for options, I agree with that. Probably we can modify this [issue](https://github.com/pravega/pravega-operator/issues/226) to address this. What do you think? @pbelgundi thanks.

Yes issue 226 is about passing JVM options to bookkeeper. For segment store and controller, it is possible to pass JVM options today, but those are one same set of options passed to both. That too needs to change. Lets address both these in PR for 226.